csharp/src/Google.Protobuf.Test/Reflection/CustomOptionsTest.cs

```diff
@@ -184,13 +184,21 @@ public void NoOptions()
             var fileDescriptor = UnittestProto3Reflection.Descriptor;
             var messageDescriptor = TestAllTypes.Descriptor;
             Assert.NotNull(fileDescriptor.CustomOptions);
+            Assert.AreEqual(fileDescriptor.GetOption(Fileopt), null);
```

Comment:
why are we intermixing the GetOption tests with a CustomOptions-related test (and all that in CustomOptionsTest.cs).
I think it would be better if GetOption methods was tested in a separate test case.

Reply:
I thought that even though the GetOption APIs weren't in a "CustomOptions" type they still worked with "custom options" so it'd be easier to test everything relating to custom options together.